CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,45 @@
+# grammY code of conduct
+
+## Who We Want to Be
+
+We as a community want to be inclusive for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, language, personal appearance, religion, or sexual identity and orientation.
+
+We are humans.
+We meet each other with respect and dignity.
+
+If it is not obvious what this means, here are a few examples of behavior that we do not tolerate.
+
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+
+## Enforcement
+
+Admins enforce our standards of acceptable behavior.
+They take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+When they take action, they will communicate this appropriately.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the admins at <https://t.me/grammyjs>.
```

Comment:
Aren't they the administrators and not the administration?

Reply:
Yeah that's just choice of words
Administrators means individual people and administration describes a group so they are supposed to report it to any member of that group (administration)

Reply:
There's no grammY administration

Reply:
Umm I think it exists unofficially the group of admins can be termed as grammy administration

Reply:
Yes.

Reply:
We need to at least replace admins with administrators. admins feels so [insert your thoughts on it here].

Reply:
Yes, and the fact that it's an unofficial group of people is intended. It's just a bunch of trustworthy folks who have been with us for a little longer. I don't want to label it and make it it's own separate thing, as if the admins were forming a body independent of “the rest.”

Reply:
We can call them the administrators if we want to avoid the abbreviation?
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the administrators at <https://t.me/grammyjs>.
```

Reply:
Yeah administrators is right

And now I think we should go forward with it, if all things have been finalised?

Reply:
> We need to at least replace admins with administrators. admins feels so [insert your thoughts on it here].

Oh I didn't even see this. Let's do it then!